build/tools/avalanche-stats.md

```diff
@@ -6,9 +6,9 @@ We have created a [stats](https://stats.avax.network) website from where you can
 
 ## What are included?
 
-We have 3 main areas in our stats: Staking, Validator Health Check and C-Chain Activity.
+We have 3 main areas in our stats: [Staking](avalanche-stats.md#staking), [Validator Health Check](avalanche-stats.md#validator-health-check) and [C-Chain Activity](avalanche-stats.md#c-chain-activity).
 
-## Staking
+## Staking <a id="staking"></a>
```

Comment:
You don't need to add this `id` to enable deep linking. Our setup get's it out of the box. When you hover the mouse over a header a deep link icon will appear and it'll correctly deep link.

For example:
<img width="181" alt="Screen Shot 2021-09-14 at 12 54 26 PM" src="https://user-images.githubusercontent.com/260614/133324903-f6b9028a-d513-47bb-96a6-23d72e176582.png">

produces this url: https://docs.avax.network/build/tools/avalanche-stats#staking

Please remove the `id`s from the `<h2>`s